src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java

```diff
@@ -65,31 +72,36 @@ public Boolean wasSoundHeard() {
   }
 
   public void verifyArticleAd() {
-    verifyFeaturedVideoElement(ARTICLE_VIDEO_PREROLL_SELECTOR, BLUE, AD_DURATION_SEC);
+    wait.forElementVisible(By.cssSelector(ARTICLE_VIDEO_PREROLL_SELECTOR), 30, 1000);
+    verifyFeaturedVideoElement(BLUE, AD_DURATION_SEC);
   }
 
   public void verifyArticleVideo() {
-    verifyFeaturedVideoElement(ARTICLE_VIDEO_SELECTOR, GREEN_OOYALA_4, VIDEO_DURATION_SEC);
+    wait.forElementVisible(By.cssSelector(ARTICLE_VIDEO_WRAPPER_SELECTOR), 30, 1000);
+    verifyFeaturedVideoElement(GREEN_OOYALA_4, VIDEO_DURATION_SEC);
   }
 
   public void verifyMobileArticleVideo() {
-    verifyFeaturedVideoElement(ARTICLE_VIDEO_MOBILE_SELECTOR, GREEN_OOYALA_3, VIDEO_DURATION_SEC);
+    wait.forElementVisible(By.cssSelector(ARTICLE_VIDEO_MOBILE_SELECTOR), 30, 1000);
+    verifyFeaturedVideoElement(GREEN_OOYALA_3, VIDEO_DURATION_SEC);
   }
 
   public void verifyLightboxAd() {
+    wait.forElementVisible(lightboxVideo);
+    System.out.println("lalalalal");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "lalalalal" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)